src/NumericHelper.php

```diff
@@ -61,4 +61,16 @@ public static function normalize($value): string
         $value = str_replace([' ', ','], ['', '.'], (string)$value);
         return preg_replace('/\.(?=.*\.)/', '', $value);
     }
+
+    /**
+     * Checks whether the given string is integer number
+     *
+     * @param string $value
+     *
+     * @return bool
+     */
+    public static function isInteger(string $value): bool
```

Comment:
```suggestion
    public static function isInteger($value): bool
```

It is convenient to be able to pass real integers here.